README.md

```diff
@@ -1 +1,137 @@
-# stencil-progress-bar
+# Stencil Progress Bar
+
+A progress bar that comes in both standard and radial flavours. 
+
+[Link to demo](#)
+
+
+
+## Requirements
+
+- Sass
+- AMD loader
+- Dust templating
+
+
+
+## Installation
+
+Install the Stencil Progress Bar using the AdaptiveJS generator:
+```
+yo adaptivejs:component --install stencil-progress-bar
+```
+You can also install interactively by running `yo adaptivejs:component` and choosing the install option.
+
+
+
+## Usage
+
+Stencil components installed with the AdaptiveJS generator require no setup. Just load the component’s template in the desired view and use the dust helper to render it. For details, see [using Components in AdaptiveJS](#).
```

Comment:
Documentation is still in beta. Not sure if we should link to the current beta docs (not sure if the URLs will change).